scripts/less2sass.js

```diff
@@ -91,9 +92,33 @@ Less2Sass.prototype.convertInterpolatedVariables = function() {
 
 Less2Sass.prototype.convertVariables = function() {
     // Matches any @ that doesn't have 'media ' or 'import ' after it.
-    var atRegex = /@(?!(media|import|mixin|font-face|keyframes)(\s|\())/g;
-
-    this.file = this.file.replace(atRegex, '$');
+    //
+    // SIMPLE LINE ICONS DEVIATION:
+    // Additionally, we capture the variable name, full variable declaration and if this
+    // should be considered a default variable for "simple-line-icons"-specific logic for
+    // creating default variables (because LESS doesn't have the concept of default variable declarations).
+    //
+    // For example, if we have a LESS variable that looked like:
+    //
+    //      @simple-line-font-path: "../fonts/"; // default
+    //
+    // The "varDeclaration" group would be 'simple-line-font-path: "../fonts/"', the varName
+    // group would be "simple-line-font-path" and the isDefault group would have content (indicating
+    // that this should be output as a default variable).
+    var varRegex = /@(?!(media|import|mixin|font-face|keyframes)(\s|\())(?<varDeclaration>(?<varName>([^:]*))[^;]*);(?<isDefault>\s*\/\/\s*default)?/g;
```

Comment:
@mruz I had to make this a capturing group which I called `isDefault` and I had to make it optional. This is to support transforming variables that aren't supposed to be default variables. For example, if we had the following:
```
@defaultVar: "value"; // default
@nonDefaultVar: "value";
```
we would want the output of:
```
$defaultVar: "value" !default;
$nonDefaultVar: "value";
```

Reply:
creative
 :D